Source/Request+AlamofireImage.swift

```diff
@@ -57,20 +97,22 @@ public final class ImageResponseSerializer: ResponseSerializer {
                                                            "image/x-ms-bmp",
                                                            "image/x-win-bitmap"]
 
-    static let streamImageInitialBytePattern = Data([255, 216]) // 0xffd8
-
     // MARK: Initialization
 
-    public init(imageScale: CGFloat = ImageResponseSerializer.deviceScreenScale,
-                inflateResponseImage: Bool = true,
+    public init(imageSerializer: ImageSerializerProtocol = ImageSerializer(),
                 emptyResponseCodes: Set<Int> = ImageResponseSerializer.defaultEmptyResponseCodes,
                 emptyRequestMethods: Set<HTTPMethod> = ImageResponseSerializer.defaultEmptyRequestMethods) {
-        self.imageScale = imageScale
-        self.inflateResponseImage = inflateResponseImage
+        self.imageSerializer = imageSerializer
         self.emptyResponseCodes = emptyResponseCodes
         self.emptyRequestMethods = emptyRequestMethods
     }
 
+    public convenience init(imageScale: CGFloat = ImageResponseSerializer.deviceScreenScale,
+                            inflateResponseImage: Bool = true) {
```

Comment:
`emptyResponseCodes` and `emptyRequestMethods` should be included as parameters as well so that the old API is still available